Source/DafnyCore/AST/Tokens.cs

```diff
@@ -224,7 +224,6 @@ public class CodeActionToken : TokenWrapper {
   public String StartLength() {
     return (line - 1) + " " + (column - 1) + " " + length;
   }
-
   public override IToken WithVal(string newVal) {
```

Comment:
I think the newline should be kept here
```suggestion

  public override IToken WithVal(string newVal) {
```